pkg/util/proc/process_unix.go

```diff
@@ -135,6 +135,9 @@ func findProcessSearch(binary, cmdlinePattern string) (*UnixProcess, error) {
 func processes() ([]*UnixProcess, error) {
 	d, err := os.Open("/proc")
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, nil
```

Comment:
why not bring in https://github.com/mitchellh/go-ps/blob/master/process_darwin.go like we did with this file?